include/http/middleware/controller.js

```diff
@@ -51,13 +51,16 @@ module.exports = pb => ({
                 ((typeof content) === 'string')) {
 
             // Add prefix for all the <a href> & <link href> tags. TO DO: be able to replace <a> href's when href is in a new line
-            content = content.replace(/(\<(?:a|link|script|img|image)(?:[^\>]|\r|\n)*\s(?:ng-href|href|src)\s*=\s*['"]\/)([^\/][^'"\>]*)(['"])/mg, function (match, p1, p2, p3) {
-                if (p2.indexOf(prefix) !== 0 && p2.indexOf(prefix) !== 1) {
-                    return `${p1}${prefix}/${p2}${p3}`;
-                } else {
-                    return `${p1}${p2}${p3}`;
-                }
-            });
+            const disableHtmlPrefix = req && req.siteObj && req.siteObj.disableHtmlPrefix;
```

Comment:
I don't think we really need another setting. Prefix should be disabled by removing its value, that's all

Reply:
Let's keep it until allegis remove the setting.